argo_qa/solrconfig.xml

```diff
@@ -33,73 +27,150 @@
       <int name="qs">1</int>
       <int name="ps">2</int>
       <float name="tie">0.01</float>
-      <!-- this qf and pf are used by default, if not otherwise specified by
-           client. The default blacklight_config will use these for the
-           "keywords" search. See the author_qf/author_pf, title_qf, etc
-           below, which the default blacklight_config will specify for
-           those searches. You may also be interested in:
-           http://wiki.apache.org/solr/LocalParams
-      -->
-       <str name="qf">
-         id
-         all_text_timv
-         active_fedora_model_ssi
-         object_type_si
-       </str>
-       <str name="pf">
-         all_text_timv^10
-       </str>
-
-      <str name="author_qf"></str>
-      <str name="author_pf"></str>
-      <str name="title_qf"></str>
-      <str name="title_pf"></str>
-      <str name="subject_qf"></str>
-      <str name="subject_pf"></str>
+      <bool name="lowercaseOperators">true</bool>
       <str name="fl"> *, score</str>
 
+      <str name="qf">
+        id
+        all_text_timv
+        object_type_ssim
+      </str>
+      <str name="pf">
+        all_text_timv^10
+      </str>
+
       <str name="facet">true</str>
       <str name="facet.mincount">1</str>
       <str name="facet.limit">10</str>
-      <str name="facet.field">active_fedora_model_ssi</str>
-      <str name="facet.field">object_type_si</str>
-
-      <str name="spellcheck">true</str>
-      <str name="spellcheck.dictionary">default</str>
-      <str name="spellcheck.onlyMorePopular">true</str>
-      <str name="spellcheck.extendedResults">true</str>
-      <str name="spellcheck.collate">false</str>
-      <str name="spellcheck.count">5</str>
-      <bool name="lowercaseOperators">true</bool>
+      <str name="facet.field">objectType_ssim</str>
+      <str name="f.objectType_ssim.facet.sort">index</str>
+      <str name="f.objectType_ssim.facet.method">enum</str>
+      <str name="facet.field">content_type_ssim</str>
+      <str name="f.content_type_ssim.facet.sort">index</str>
+      <str name="f.content_type_ssim.facet.method">enum</str>
+      <str name="facet.field">nonhydrus_apo_title_ssim</str>
+      <str name="facet.field">released_to_earthworks_dtsi</str>
+      <str name="facet.field">released_to_searchworks_dtsi</str>
     </lst>
-    <arr name="last-components">
-      <str>spellcheck</str>
-    </arr>
   </requestHandler>
 
-  <requestHandler name="/argosearch" class="solr.SearchHandler"> <!-- experimental/useless -->
+  <!-- for relevancy experiments -->
+  <requestHandler name="/qttest" class="solr.SearchHandler">
```

Comment:
renamed and repurposed this request handler.   The current incarnation is to try out some new relevancy tweaks.